.github/workflows/cherry-pick.yml

```diff
@@ -62,17 +62,17 @@ jobs:
           jq -r '.migrations | map(select(.name != "inject-flags")) | map(select(.name != "test-bed-get")) | {"migrations": .}' migrations.json >  migrations.json.tmp \
             && mv migrations.json.tmp migrations.json
 
-          # Run the migrations and fix any linting and formatting issues.
-          npx nx migrate --run-migrations --if-exists \
-            && npx nx affected -t lint --fix \
-            && npx nx format:write
+          # Run the migrations and fix any formatting issues.
+          npx nx migrate --run-migrations --if-exists && npx nx format:write
 
           if [ $? -ne 0 ]; then
             echo "CHERRY_PICK_RESULT=failed" >> $GITHUB_ENV
             exit 0
           fi
 
           rm -f ./migrations.json
+          # Any changes to package.json and package-lock.json should not be committed.
+          git restore package.json package-lock.json
```

Comment:
The migration runs `npm install` and makes unnecessary changes to `package-lock.json`, making it invalid. This step should not introduce package changes without further review.